lib/commands/plugin-add.sh

```diff
@@ -28,7 +35,7 @@ plugin_add_command() {
     display_error "Plugin named $plugin_name already added"
     exit 1
   else
-    if ! git clone "$source_url" "$plugin_path"; then
+    if git clone -b "$plugin_branch" "$source_url" "$plugin_path"; then
```

Comment:
I guess you removed the `!` un-intentionally, the script should fail if it's not able to clone the repo.